README.md

```diff
@@ -652,6 +653,8 @@ Sample result:
 }
 ```
 
+Note the `last_modified` value which represents the collection's last modified timestamp.
```

Comment:
* `collection current timestamp`
* Add link to http://kinto.readthedocs.org/en/latest/api/1.x/cliquet/timestamps.html